pkgs/development/compilers/gcc/12/default.nix

```diff
@@ -342,4 +352,13 @@ stdenv.mkDerivation ({
 }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
-)
+))
+(callPackage ../common/external-bootstrap.nix {
```

Comment:
fyi
https://github.com/NixOS/nixpkgs/pull/215751
https://github.com/NixOS/nixpkgs/pull/215156

Reply:
#215156 is totally awesome, thank you so much.